target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -131,7 +131,8 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan" "2:lan" "3:lan" "4:lan" "5:wan" "6@eth1"
 		;;
-	compex,wpj344-16m)
+	compex,wpj344-16m|\
+	compex,wpj563)
```

Comment:
ar71xx has 2:lan 3:wan for this one?

Reply:
i've double-checked the port mapping using swconfig. port 3 is next to the LEDs (lan in stock), port 2 is next to the DC socket (wan in stock).

in the official openwrt-sdk (not to be mistaken with osdk), it is the other way around:
https://github.com/compex-systems/lede/blob/openwrt-18.06/target/linux/ar71xx/base-files/etc/board.d/02_network#L538

i find the stock solution (wan next to the power jack) more intuitive but i don't mind either way.

Reply:
I prefer the stock solution as well. However, that means that ar71xx has the "wrong" port assignment right now?

Reply:
"wrong" is a harsh term :-D
i've just noticed compex uses [mehlis code](https://github.com/compex-systems/lede/blame/openwrt-18.06/target/linux/ar71xx/base-files/etc/board.d/02_network#L538) for their openwrt SDK.

i'll do a third check with compexWRT/stock/OSDK (have only one board here).

Reply:
But since this is just a "board", there are no external indicators and the choice of LAN vs. WAN is conceptually arbitrary in any case? So, we might just decide by ourselves what is right here, vendor firmware or previous established use in ar71xx?

Reply:
Still waiting with further steps for an answer to this "question" ...

Reply:
i'm very sorry to have kept you waiting. at one point last week i was very unsure about which firmware i tested.
the information about stock port assignment i've given above is wrong.

both ports are lan ports in stock (like with the wpj428).
i still prefer the more intuitive (current) port assignment.

tested using this firmware:
http://downloads.compex.com.sg/?dir=uploads/CompexWRT/Stable/v6.2.12-b190718-OEM

```
14: br-lan: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default qlen 1000
    link/ether 04:f0:21:3d:ec:71 brd ff:ff:ff:ff:ff:ff
15: eth0.1@eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master br-lan state UP mode DEFAULT group default qlen 1000
    link/ether 04:f0:21:3d:ec:71 brd ff:ff:ff:ff:ff:ff
16: eth0.2@eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master br-lan state UP mode DEFAULT group default qlen 1000
    link/ether 04:f0:21:3d:ec:71 brd ff:ff:ff:ff:ff:ff
```